svg-icons-builder/constants/index.ts

```diff
@@ -0,0 +1,28 @@
+import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect';
+import { JsonObject } from '@angular-devkit/core';
+import { ConstantsConversionOptions, ConversionType, convertToConstants, mergeWithDefaults } from 'svg-to-ts';
+
+interface Options extends ConstantsConversionOptions {}
+
+// Using `Options & JsonObject` instead of extending JsonObject because of optional boolean
+export default createBuilder<Options & JsonObject>((options: Options, context: BuilderContext) => {
+  return new Promise<BuilderOutput>(async (resolve, reject) => {
+    try {
+      if (options.conversionType !== ConversionType.CONSTANTS) {
```

Comment:
I remove the check for `options.conversionType` because schema now requires it and it won't get to the error.
I added a check to match builder with the `options.conversionType` and removed extra conditionals to simplify.

Reply:
Perfect - that's also how I imagined it.

Reply:
With the comment above I think that we can also remove this check. Basically, we would remove the `conversionType` from the options in this project.

Reply:
Gave this a shot and it raises a question. It's trivial to remove the validation bit in the `index.ts` files. The side effect is I can't remove the requirement from `schema.json` because that drives the conditional logic and schema checks run before any typescript checks, otherwise I could just for the correct value on the typescript side.

Removing validation would be ok, maybe, because in an ideal world the ng-add setup the builder correctly and it should work. Other fields would also likely fail if the user mismatched them, just not sure they would know why.

Thoughts on this?

Reply:
I see. Very good point. I didn't think about this. In this case, I would leave it as it is. Maybe it's even better because then the options are the same as in `svg-to-ts`